packages/amplify-category-function/src/events/postEnvRemoveHandler.ts

```diff
@@ -1,8 +1,18 @@
+import { printer } from '@aws-amplify/amplify-prompts';
 import { $TSContext } from 'amplify-cli-core';
 import { FunctionSecretsStateManager } from '../provider-utils/awscloudformation/secrets/functionSecretsStateManager';
 
 export const postEnvRemoveHandler = async (context: $TSContext, envName: string) => {
-  await removeAllEnvSecrets(context, envName);
+  try {
+    await removeAllEnvSecrets(context, envName);
+  } catch (err) {
+    // catching this silently because that was the previous behavior
+    // previously this error was caught by the platform event firing logic and silently ignored
+    // now we are ignoring errors at the individual event handler
+    printer.debug(`function category postEnvRemoveHandler failed to run.`);
+    printer.debug(`You may need to manually clean up some function state.`);
```

Comment:
Do we need to throw a amplify Error with resolution or doc link on how to cleanup ?

Reply:
This case was previously ignored, so this matches the existing behavior but with some additional debug lines